LOCALAPP.md

```diff
@@ -91,28 +91,33 @@ Currently we have not been able to provide a 100% common setup between Windows a
      ```shell
      sudo nginx -s reload
      ```
-3. Set path to app folder in local platform services:
-   - Open `appSettings.json` in the `LocalTest` folder, f.ex. in Visual Studio Code
-     ```shell
-     cd LocalTest
-     code appSettings.json
-     ```
-   - Change the setting `AppRepositoryBasePath` to the parent folder where you have your application repos (`C:/repos/` as an example) as  to your app on the disk.
+3. Set path to app folder in local platform services. There are two ways to do this:
```

Comment:
Is the reasoning for not doing this in the appsettings.json to allow the developers to have their own path without having to change appsettings.json?

Reply:
Yes, the issue with changing the file is that we later risk commiting them when working on LocalTest. There is also the issue of pulling updates with local changes. Git either stops you or overwrite your changes.

Reply:
When LocalTest runs in docker for most users, this trick will no longer be required.